16feb/about.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Welcome To About</title>
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div class="container">
+                <h1 class="logo">HBT</h1>
+                <ul>
+                    <li><a href="index.html"> HOME</a></li>
+                    <li><a class="current" href="about.html">ABOUT </a></li>
+                    <li><a href="contact.html">CONTACT</a></li>
+                </ul>
+            </div>
+        </nav>
+
+    </header>
+    <section id="about-info" class="py-3">
+        <div class="container">
+            <div class="info-left">
+                <h1 class="l-heading"><span class="primary-heading">About</span> Hotel BT</h1>
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+                    Ea odit reiciendis alias delectus eveniet debitis amet aut aliquam
+                    sunt labore? Dolor oditv enim consequatur fuga laborum, ratione consequuntur nam quasi!</p>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore, natus vitae nemo tempora
+                    esse repellendus ab enim laudantium distinctio sed.</p>
+            </div>
+
+            <div class="info-right">
+                <img src="./img/about info.jpg" alt="">
```

Comment:
Try to not put spaces between file names, either use 
file_name(snake_casing), or fileName(camelcasing).